daffodil-test/src/test/scala/org/apache/daffodil/section00/general/TestDefError.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.section00.general
+
+import org.apache.daffodil.tdml.Runner
+import org.junit.{AfterClass, Test}
+
+object TestDefError {
+
+  val testDir = "/org/apache/daffodil/section00/general/"
+  val runner = Runner(testDir, "generalSchedmaDefinitionError.tdml",validateTDMLFile = false)
```

Comment:
In addition to changing the test name as Steve suggested, I suggest also changing the scala and tdml names.  I'm fine with a shorter name than "dfdl_format_prefixed_attributes_sde" or a camel case name, just make the name more descriptive/specific.  Please also pay attention to Scala's [style guide](https://docs.scala-lang.org/style/), e.g., each comma in the above line should have a space character after it.